packages/pwa-kit-react-sdk/scripts/upload.js

```diff
@@ -9,7 +9,7 @@ const Utils = require('./utils')
 const buildRequest = require('./build-request')
 
 const ARCHIVE = 'build.tar'
-const DEFAULT_ORIGIN = process.env.CLOUD_API_BASE || 'https://cloud.mobify.com'
+const DEFAULT_ORIGIN = process.env.CLOUD_API_BASE || 'https://runtime.commercecloud.com/'
```

Comment:
this will break because the API is only working on the `https://cloud.mobify.com` domain. 

`runtime.commercecloud.com` is just the react app.

Reply:
I reverted back. Thanks.